style.css

```diff
@@ -370,6 +370,128 @@
   padding-top: 25px;
 }
 
+.form-section {
+  margin-top: 70px;
+  width: 100%;
+  height: 750px;
+  background-color: white;
+}
```

Comment:
- [ ] As the project requirements suggest, please try as much as possible to ensure you use flexbox to layout the elements of the contact form. I noticed that you used flex box in the desktop version which is very good approach 👍. However, since our focus is on mobile version of the portifolio kindly ensure to use flexbox to layout the elements ☺️.